shared/cfg/guest-os/Linux/RHEL/8.cfg

```diff
@@ -0,0 +1,50 @@
+- 8:
+    variants:
+        - aarch64:
+            vm_arch_name = aarch64
+        - ppc64:
+            vm_arch_name = ppc64
+        - ppc64le:
+            vm_arch_name = ppc64le
+        - s390x:
+            vm_arch_name = s390x
+        - x86_64:
+            vm_arch_name = x86_64
+    os_variant = rhel8
+    unattended_install.url:
+        #Running from http server, RHEL requires more memory
+        mem = 2048
+    nic_hotplug:
+        modprobe_module =
+    block_hotplug:
+        modprobe_module =
+    no unattended_install..floppy_ks
+    unattended_install, svirt_install:
+        kernel = images/${os_variant}-${vm_arch_name}/vmlinuz
+        initrd = images/${os_variant}-${vm_arch_name}/initrd.img
+        unattended_install, check_block_size.4096_512, check_block_size.512_512, svirt_install:
+            cdrom_unattended = images/${os_variant}-${vm_arch_name}/ks.iso
+            syslog_server_proto = udp
+        # ARCH dependent things
+        aarch64:
+            grub_file = /boot/grub/grub.conf
+            install_timeout = 7200
+        ppc64:
```

Comment:
RHEL8 doesn't support ppc64, you can remove it here.
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8-beta/html-single/8.0_beta_release_notes/index#architectures

Reply:
@PaulYuuu corrected, thanks.